__tests__/hooks/useAuth.test.tsx

```
@@ -849,4 +954,482 @@ describe('useAuth Hook', () => {
+      json: () => Promise.resolve({
+        authProcess: 'azure',
+        isAuthDisabled: false,
+        autoLogin: false,
+      }),
+    } as Response);
+
+    // Mock signOut to fail to trigger fallback
+    mockSignOut.mockRejectedValue(new Error('SignOut failed'));
+
+    // Mock performCompleteLogoutCleanup to also fail
+    const { performCompleteLogoutCleanup } = require('@/lib/utils/token-cleanup');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)